### PR TITLE
2020 Data Added

### PR DESCRIPTION
**Pull request by pholleway**
Before you submit a pull request, please fill in the following:

**Related Issue(s):**
Adding newly available 2020 data. 

**PR checklist:**

- [x] Code is formatted (run `scripts/format`).
- [x] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).
- [x] Documentation has been updated to reflect changes, if applicable.
- [x] Examples are added to `examples/`

### CODE REVIEW

File: item.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
+{
```
Comment (gadomski) [inline item.json]: This file should not live at the top-level of the repository ... is it meant to be an example?

File: src/stactools/drcog_lulc/constants.py [hunk 2]
```diff
@@ -13,12 +13,17 @@
...
+        "drcog-lulc-2020"
+    }
+}
+ITEM_DESCRIPTION = "DRCOG LULC at 1m resolution"
 MISSION = "DRCOG LULC"
 EPSG = 26913
-START_TIME = datetime.datetime.fromisoformat("2018-01-01T00:00:00+00:00")
-END_TIME = datetime.datetime.fromisoformat("2018-12-31T23:59:59+00:00")
 CLASSIFICATION_SCHEMA = (
     "https://stac-extensions.github.io/classification/v1.0.0/schema.json"
```
Comment (gadomski) [inline src/stactools/drcog_lulc/constants.py]: This schema should be updated to the latest version, https://stac-extensions.github.io/classification/v1.1.0/schema.json.

**Review by gadomski — CHANGES_REQUESTED** [review]

File: src/stactools/drcog_lulc/constants.py (hunk 2 above)
Comment (pholleway) [reply src/stactools/drcog_lulc/constants.py]: updated

File: item.json (hunk 1 above)
Comment (pholleway) [reply item.json]: Not sure how it ended up at the top-level. There was already an item file in the examples folder. I removed the "item.json" from the top-level.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gadomski) [inline .vscode/settings.json]: This file should be removed, we don't need editor-specific files in the repo.

**Review by gadomski — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 3 above)
Comment (pholleway) [reply .vscode/settings.json]: removed

**Review by gadomski — APPROVED** [review]

**Pull request merged**